packages/dashboard-message-bus-common/lib/messages.ts

```diff
@@ -66,6 +78,12 @@ export interface InvalidateMessage extends Message {
   payload: number;
 }
 
+export const isCliEventMessage = (
+  message: Message
+): message is CliEventMessage => {
```

Comment:
Can you make this generic to `T` the same way `CliEventMessage<T>` is?

Even if we have to specify a default value, probably still worth keeping the parameter from being obscured by this type guard.